test/test_double_buffering.cpp

```diff
@@ -0,0 +1,686 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+
+#include <ops/all_ops.h>
+#include <test/utils.h>
+#include <test/validator.h>
+
+namespace nvfuser {
+
+namespace {
+class DoubleBufferingTest : public NVFuserTest {};
+} // anonymous namespace
+
+TEST_F(DoubleBufferingTest, FusionDoubleBuffering1) {
```

Comment:
nit: remove the `Fusion` prefix in test name

Reply:
Sure, I will update the PR and then integrate it.

Thanks!